.travis.yml

```diff
@@ -0,0 +1,51 @@
+# The language in this case has no bearing - we are going to be making use of conda for a
+# python distribution for the scientific python stack.
+language: generic
+
+sudo: false
+
+env:
+    global:
+       - CONDA_INSTALL_LOCN="${HOME}/conda"
+
+       # Adds a GH_TOKEN (conda-forge-admin's conda-forge.github.io token)
+       - secure: __FIX_ME__
```

Comment:
We need to add a token here. FWICT it is going to be different than the old one as it will have to be encrypted for this repo instead of the webpage repo.